test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -204,6 +204,7 @@ def allow_listed(schema, allow_list):
 dont_parse_list = [
     ("_TorchScriptTesting.*", datetime.date(2099, 9, 17)),
     ("test_backend", datetime.date(2099, 9, 17)),
+    ("c10d.frontend", datetime.date(2020, 12, 30)),
```

Comment:
curious why we need this?

Reply:
Previously, I made the mistake of putting frontend into `c10d` namespace rather than `dist_c10d`, which all of our other torchbinds are in. So I changed the namespace in this diff, resulting in a BC-breaking change.